rero_ils/modules/cli/utils.py

```
@@ -188,6 +189,81 @@ def check_json(paths, replace, indent, sort_keys, verbose):
+    """
+    with open(file_name) as opened_file:
+        header = "".join(itertools.islice(opened_file, 5))
+    return all(tag in header for tag in SPDX_TAGS)
+
+
+@utils.command("check_license")
+@click.option("-v", "--verbose", "verbose", is_flag=True, default=False)
+def check_license(verbose):
+    """Check SPDX license headers of the files tracked by git."""
+    click.secho("Testing SPDX license headers.", fg="green")
+    tracked_files = subprocess.check_output(["git", "ls-files", "-z"], text=True).split("\0")
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)